### PR TITLE
Pcntl php84

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by devnexen** [comment]
for #3872

File: reference/pcntl/constants.xml [hunk 1]
```diff
@@ -1502,6 +1526,57 @@
...
+    (<type>enum</type>)
+   </term>
+   <listitem>
+    <simpara>
+     Runs the process at high priority level but below Pcntl::UserInteractive ones.
+     Available as of PHP 8.4.0 (macOs only).
+    </simpara>
+   </listitem>
+  </varlistentry>
+ </variablelist>
```
Comment (Girgias) [inline reference/pcntl/constants.xml:1579]: This is an enum, so it should not be documented here actually see #4080 for how such a page should look (albeit we don't have rendering support yet).

**Review by Girgias — CHANGES_REQUESTED** [review]

File: reference/pcntl/constants.xml (hunk 1 above)
Comment (devnexen) [reply reference/pcntl/constants.xml:1579]: Yes I expected you would ask me changes, kept it as a separate commit. Ok will have a look.

Comment (Girgias) [reply reference/pcntl/constants.xml:1579]: Feel free to merge the first commit then :)

**Pull request merged**